apps/aecontract/src/aect_contracts.erl

```diff
@@ -142,11 +125,7 @@ serialize(#contract{} = C) ->
       ?CONTRACT_TYPE,
       ?CONTRACT_VSN,
       serialization_template(?CONTRACT_VSN),
-      [ {pubkey, pubkey(C)}
```

Comment:
Could you please make the environments start from a fresh DB, by [bumping the DB folder](https://github.com/aeternity/epoch/blob/10b0a4e9a03d178b0d6fed470d7b246b2ae876c0/deployment/ansible/deploy.yml#L69)?

Reply:
Somehow lost in rebase?